apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx

```diff
@@ -13,6 +22,29 @@ export default class ModerationNotification extends Component {
     )
   }
 
+  async switchIsPending () {
```

Comment:
just a little note here, because i saw it in other spots already.

we should only use `async` if we use `await` in the function body. otherwise we are turning this function into a asynchronous one, but not really use it. so either `Promise: then` or `async/await`.

we could include a rule in eslint:
https://eslint.org/docs/rules/require-await